tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js

```diff
@@ -73,8 +73,8 @@ module.exports = {
      * @param {boolean} expectToSucceed
      * @return {*}
      */
-    rename: function(toName, expectToSucceed = true) {
-      this.initAjaxCounters()
+    rename: async function(toName, expectToSucceed = true) {
```

Comment:
You already had this for `rename` in your other PR. Were you aiming for making `delete` async?

Reply:
this commit is only there because I branched off #3944, see original comment

Reply:
that commit is gone now

Reply:
👍